HyperionRGB/WrapperWebconfig.h

```diff
@@ -2,7 +2,7 @@
 #define WrapperWebconfig_h
 #include "BaseHeader.h"
 
-#include <ESP8266WebServer.h>
+#include <ESP32WebServer.h>
```

Comment:
If you include the ESP32 like this and these changes are merged back, I'll assume it will only be working with ESP32 and not anymore with ESP8266. Can you make it configurable via the ConfigStatic.h which board you are using? Similar to the define's for the PIN_ORDERs

Reply:
Did it as compiler directive (see devlop branch"